src/main/java/com/jalasoft/jfc/model/ImageParam.java

```diff
@@ -15,13 +16,20 @@
  * @author Oscar Lopez.
  * */
 public class ImageParam {
-    private String inputPathFile;   //Variable for obtain input path of Image.
-    private String outputPathFile;  //Variable of output path of Image.
-    private String outputPathThumbnail; //Variable of output path of Thumbnail of a Image.
-    private int widthOfFile;    //Variable for change the width of Image.
-    private int heightOfFile;   //Variable for change the height of Image.
-    private int whiteBlankPercentage;   //Variable for change the percentage of white and black of a Image.
-    private double degreesToRotate; //Variable for rotate a Image.
+    //Variable for obtain input path of Image.
+    private String inputPathFile;
+    //Variable of output path of Image.
+    private String outputPathFile;
+    //Variable of output path of Thumbnail of a Image.
+    private String outputPathThumbnail;
+    //Variable for change the width of Image.
+    private int widthOfFile;
+    //Variable for change the height of Image.
+    private int heightOfFile;
+    //Variable for change the percentage of white and black of a Image.
+    private int whiteBlankPercentage;
+    //Variable for rotate a Image.
+    private double degreesToRotate;
```

Comment:
Here you can add the variables documentation at right side of each variable, but only until to 80 characters per line